pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <groupId>com.facebook.presto</groupId>
     <artifactId>presto-root</artifactId>
-    <version>0.149-tw-0.27</version>
+    <version>0.149-tw-0.28</version>
```

Comment:
This PR should not add functionality and rev the presto version at the same time. We should rev versions in PRs of dedicated to just that.